django/db/models/functions.py

```diff
@@ -44,23 +45,22 @@ def __init__(self, left, right, **extra):
     def as_sqlite(self, compiler, connection):
         coalesced = self.coalesce()
         coalesced.arg_joiner = ' || '
-        coalesced.template = '%(expressions)s'
-        return super(ConcatPair, coalesced).as_sql(compiler, connection)
+        return super(ConcatPair, coalesced).as_sql(compiler, connection, template='%(expressions)s')
 
```

Comment:
This tells me that the ``arg_joiner`` is also backend-dependant. Should we also use this as kwarg in ``Func``-class?